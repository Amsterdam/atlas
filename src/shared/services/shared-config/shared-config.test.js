import SHARED_CONFIG from './shared-config'

describe('The sharedConfig service', () => {
  it('gives you the configuration based on global environment', () => {
    expect(SHARED_CONFIG).toMatchSnapshot()
  })
})
