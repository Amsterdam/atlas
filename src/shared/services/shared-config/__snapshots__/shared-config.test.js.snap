// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The sharedConfig service gives you the configuration based on global environment 1`] = `
Object {
  "API_ROOT": "https://acc.api.data.amsterdam.nl/",
  "AUTH_HEADER_PREFIX": "Bearer ",
  "CACHE_THRESHOLD": 30,
  "CMS_ROOT": "https://acc.cms.data.amsterdam.nl/",
  "PANORAMA_THUMBNAIL_URL": "panorama/thumbnail/",
  "RADIUS": 50,
  "ROOT": "https://acc.data.amsterdam.nl/",
  "ROOT_KEYS": Array [
    "API_ROOT",
  ],
  "THUMBNAIL_WIDTH": 240,
}
`;

exports[`The sharedConfig service gives you the configuration based on global environment 2`] = `
Object {
  "API_ROOT": "https://acc.api.data.amsterdam.nl/",
  "AUTH_HEADER_PREFIX": "Bearer ",
  "CACHE_THRESHOLD": 30,
  "CMS_ROOT": "https://acc.cms.data.amsterdam.nl/",
  "PANORAMA_THUMBNAIL_URL": "panorama/thumbnail/",
  "RADIUS": 50,
  "ROOT": "https://acc.data.amsterdam.nl/",
  "ROOT_KEYS": Array [
    "API_ROOT",
  ],
  "THUMBNAIL_WIDTH": 240,
}
`;

exports[`The sharedConfig service gives you the configuration based on global environment 3`] = `
Object {
  "API_ROOT": "https://acc.api.data.amsterdam.nl/",
  "AUTH_HEADER_PREFIX": "Bearer ",
  "CACHE_THRESHOLD": 30,
  "CMS_ROOT": "https://acc.cms.data.amsterdam.nl/",
  "PANORAMA_THUMBNAIL_URL": "panorama/thumbnail/",
  "RADIUS": 50,
  "ROOT": "https://acc.data.amsterdam.nl/",
  "ROOT_KEYS": Array [
    "API_ROOT",
  ],
  "THUMBNAIL_WIDTH": 240,
}
`;

exports[`The sharedConfig service gives you the configuration based on global environment 4`] = `
Object {
  "API_ROOT": "https://acc.api.data.amsterdam.nl/",
  "AUTH_HEADER_PREFIX": "Bearer ",
  "CACHE_THRESHOLD": 30,
  "CMS_ROOT": "https://acc.cms.data.amsterdam.nl/",
  "PANORAMA_THUMBNAIL_URL": "panorama/thumbnail/",
  "RADIUS": 50,
  "ROOT": "https://acc.data.amsterdam.nl/",
  "ROOT_KEYS": Array [
    "API_ROOT",
  ],
  "THUMBNAIL_WIDTH": 240,
}
`;
